src/unstract/sdk/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -26,21 +26,17 @@
       "format": "password",
       "description": "API key obtained from the Unstract developer portal (https://unstract-api-resource.developer.azure-api.net)"
     },
-    "processing_mode": {
+    "mode": {
       "type": "string",
-      "title": "Processing Mode",
+      "title": "Mode",
       "enum": [
-        "text",
-        "ocr"
+        "native_text",
```

Comment:
Should we change the underscores to hyphens for consistency? @johnyrahul

Reply:
This will introduce additional transformations at the unstract SDK side to the LLM whisperer. I think it is ok to proceed with the same.